/**: # wildcard for the node name
  ros__parameters:
    x0: [1.0, 2.0, -0.1, 0.0, 0.0, 0.0]
    range_noise_stddev: 0.1   # in m
    process_noise_position_stddev: 0.5  # in m
    r_tag0_K: [0.7, 3.8, -0.5]
    r_tag1_K: [1.3, 3.8, -0.5]
    r_tag2_K: [0.7, 3.8, -0.9]
    r_tag3_K: [1.3, 3.8, -0.9]
    initial_state_covariance: 2.0
    pressure_noise_stddev: 0.01
    atmospheric_pressure: 100900
    r_pressure_R: [0.0, 0.0, 0.0]
    rho: 1000
    g: 9.81
    r_Cam_R: [0.0, 0.0, 0.0]
    x_limits: [2.0, 4.0, -1.5]